pkg/build.sh

```
@@ -0,0 +1,73 @@
+
+rm -rf "${LIBS_PATH}/server/__pycache__"
+rm -rf "${LIBS_PATH}/server/mem_agent/__pycache__"
+rm -rf "${LIBS_PATH}/server/backend/__pycache__"
+rm -rf "${LIBS_PATH}/server/.venv"
+rm -rf "${LIBS_PATH}/server/stack"
+
+cp -r "${MODELFILE_DIR}" "${LIBS_PATH}"
+
+
+# Creating .pkg
+pkgbuild --root pkgroot --scripts pkg/scripts --identifier com.tilesprivacy.tiles --version "$VERSION" "tiles-${VERSION}".pkg"
```

Shellcheck (0.11.0)

[error] Couldn't parse this double quoted string. Fix to allow more checks.

(SC1073)